tests/routers/test_job_manager.py

```diff
@@ -589,3 +589,60 @@ def test_set_job_status_with_invalid_job_id(normal_user_client: TestClient):
     # Assert
     assert r.status_code == 404, r.json()
     assert r.json() == {"detail": "Job 999999999 not found"}
+
+
+def test_delete_job(normal_user_client: TestClient):
+    # Arrange
+    job_definitions = [TEST_JDL]
+    r = normal_user_client.post("/jobs/", json=job_definitions)
+    assert r.status_code == 200, r.json()
+    assert len(r.json()) == 1
+    job_id = r.json()[0]["JobID"]
+
+    # Act
```

Comment:
Perhaps this would make sense as a pytest fixture to remove the repetition?

```suggestion
def test_delete_job(normal_user_client: TestClient, test_job_id: int):
```